libhttpserver/server.go

```diff
@@ -92,7 +92,7 @@ func (e obsoleteTrackingFS) isObsolete() bool {
 func (s *Server) getHTTPFileSystem(ctx context.Context, requestPath string) (
 	toStrip string, fs http.FileSystem, err error) {
 	fields := strings.Split(requestPath, "/")
-	if len(fields) < 3 {
+	if len(fields) < 2 {
 		return "", nil, errors.New("bad path")
 	}
 
```

Comment:
won't this make `fields[1]` below include the query parameters?

Reply:
No, because this is `request.URL.Path` which is already parsed and doesn't include the query part.